### PR TITLE
Apply rustfmt

### PR DESCRIPTION
**Pull request by ebfull**
Supersedes #76 

I've made a change to the CI which checks `rustfmt` against the code. Let's see if it worked.

### CODE REVIEW

**Comment by ebfull** [comment]
@bmerge try

**Comment by bmerge** [comment]
:hourglass: Trying commit 314b7d1b2110b58bdd48a314c90e4cd37484ff3c with merge 7bbbacad776eed279c25a0920fdf68c6a08ab5a2...

**Comment by bmerge** [comment]
:broken_heart: Test failed - [pairing-linux64-try](http://ci.bellman.rs:8010/builders/pairing-linux64-try/builds/5)

**Comment by ebfull** [comment]
@bmerge try

**Comment by bmerge** [comment]
:hourglass: Trying commit d07b6aed43b6a3cb2dea0e710a5a383aa6ee9cce with merge 54a7fa882fcd84bab17f17329af89a63feb79c7a...

**Comment by bmerge** [comment]
:broken_heart: Test failed - [pairing-linux64-try](http://ci.bellman.rs:8010/builders/pairing-linux64-try/builds/6)

**Comment by ebfull** [comment]
Worked, this is ready to go.

I haven't configured rustfmt to do anything differently for our codebase, in line with @nathan-at-least's desire.

File: src/bls12_381/ec.rs [hunk 1]
```diff
@@ -1326,29 +1540,36 @@ pub mod g2 {
...
-                                             0x91d50792876a202, 0x5d543a95414e7f1]);
+            let cofactor = BitIterator::new([
+                0xcf1c38e31c7238e5,
+                0x1616ec6e786f0c70,
+                0x21537e293a6691ae,
+                0xa628f1cb4d9e82ef,
+                0xa68a205b2e5a7ddf,
+                0xcd91de4547085aba,
+                0x91d50792876a202,
+                0x5d543a95414e7f1,
```
Comment (daira) [inline src/bls12_381/ec.rs]: 0x091d50792876a202, etc.

File: src/bls12_381/fq.rs [hunk 2]
```diff
@@ -1238,9 +2558,33 @@ fn test_fq_sub_assign() {
...
+        0x329300000030ffcf,
+        0x633c00c02cc40028,
+        0xbef70d925862a942,
+        0x4f7fa2a82a963c17,
+        0xdf1eb2575b8bc051,
+        0x1162b680fb8e9566,
+    ])));
+    assert!(
+        tmp
+            == Fq(FqRepr([
```
Comment (daira) [inline src/bls12_381/fq.rs]: This is a strange layout. I guess ``tmp`` could have been longer.

**Comment by ebfull** [comment]
I'm timing out @str4d's ACK and accepting me/daira as sufficient. I tested this locally again to make sure the changes from `rustfmt` match exactly by applying it to `src/lib.rs` on `master`/`c9cacc74672e3406013f8f1f8c04ed865ab36f9e`.

@bmerge r+

**Comment by bmerge** [comment]
:pushpin: Commit 53083f4 has been approved by `ebfull`

**Comment by bmerge** [comment]
:hourglass: Testing commit 53083f4290c3fd4967285677e35ebd24d2f4f4ae with merge 6fea7e382d6327164996f28bd9a6b092df9eb18d...

**Comment by bmerge** [comment]
:sunny: Test successful - [pairing-linux32-auto](http://ci.bellman.rs:8010/builders/pairing-linux32-auto/builds/50), [pairing-linux64-auto](http://ci.bellman.rs:8010/builders/pairing-linux64-auto/builds/55), [pairing-windows32msvc-auto](http://ci.bellman.rs:8010/builders/pairing-windows32msvc-auto/builds/50), [pairing-windows64msvc-auto](http://ci.bellman.rs:8010/builders/pairing-windows64msvc-auto/builds/50)
Approved by: ebfull
Pushing 6fea7e382d6327164996f28bd9a6b092df9eb18d to master...

**Pull request merged**